### DISCUSSION TITLE
Help with setting up Artisan with MAX6675 and ARDUINO UNO

### DISCUSSION THREAD

**Discussion by DanielFoster66** [artisan-roaster-scope/artisan#1757] [Q&A]
Hi there, 

Hope one of you can help me get Artisan to read temperatures from my MAX6675. I get the temperatures to show on the IDE in the serial monitor, but then get this error when trying to connect via the Arduino TC4 device setting: 

"ser.ARDUINOTC4temperature): Arduino could not set channels@line 6239"

I then tried a TC4 emulator and MODBUS but just got error after error. 

Please help!

Thanks :)

**Comment by beanoccio** [comment]
With a little research you can find working instructions online.
You should post your Artisan settings and your Arduino sketches.
